build.sbt

```diff
@@ -3,16 +3,18 @@ import com.typesafe.sbt.SbtGhPages.GhPagesKeys._
 
 lazy val buildSettings = Seq(
   organization := "io.github.finagle",
-  version := "0.4.3",
-  scalaVersion := "2.12.2",
-  crossScalaVersions := Seq("2.11.11","2.12.2")
+  version := "17.11.0",
```

Comment:
Let's make this `0.5.0` - I don't think we have to follow the same version numbering of Finagle. Major version 17 implies a level of stability that I'm not sure I'd assert for finagle-postgres.

Other than that, LGTM!